hazelcast/src/hazelcast/client/spi/impl/AbstractClientInvocationService.cpp

```diff
@@ -279,15 +279,11 @@ namespace hazelcast {
 
                 void AbstractClientInvocationService::ResponseThread::process(
                         const AbstractClientInvocationService::ClientPacket &packet) {
```

Comment:
Can you completely remove the packet and pass only ClientMessage here ?

Reply:
Ok, removed ClientPacket and used ClientMessage.